tesseract/tesseract_collision/src/bullet/tesseract_gjk_pair_detector.cpp

```diff
@@ -821,10 +821,10 @@ void TesseractGjkPairDetector::getClosestPointsNonVirtual(const ClosestPointInpu
     }
 
     m_simplexSolver->reset();
-    if (status == 0 && !m_cdata->req.calculate_distance)
+    if (status == 0 && !m_cdata->req.calculate_penetration)
```

Comment:
This was a bug. There is a following check that handles if distance is disabled.